core/store/tx_manager_test.go

```diff
@@ -945,6 +945,8 @@ func TestTxManager_WithdrawLink_Unconfigured_Oracle(t *testing.T) {
 }
 
 func TestManagedAccount_GetAndIncrementNonce_YieldsCurrentNonceAndIncrements(t *testing.T) {
+	t.Parallel()
+
```

Comment:
Anyway we could avoid this?  Could we revisit our policy on parallelizing unit tests?